packages/core/src/eth/wallet.ts

```diff
@@ -626,9 +625,8 @@ export class EthUserWallet extends EthLikeWallet {
     this.userWalletData = userWalletData;
   }
 
-  async getNonce(): Promise<BN> {
-    const nonce = await this.client.get<NonceDTO>(`${this.baseUrl}/nonce`);
-    return BNConverter.hexStringToBN(String(nonce.nonce));
+  getNonce(): BN {
+    return BNConverter.hexStringToBN("0x" + randomBytes(32).toString("hex"));
   }
```

Comment:
Same function with super class, this definition is redundant.

Reply:
Ok!